tests/pos/t284-pos.scala

```diff
@@ -1,5 +1,5 @@
 trait B[T] {
-    def f1(a: T): Unit { }
+    //def f1(a: T): Unit { }
```

Comment:
See the commit message (in fact, I encourage you to read all the commit messages as they'll help you understand what each change is for): https://github.com/smarter/dotty/commit/968bc19929c0271dfd74753ba6f4740e0cede014